.tools/validate.go

```diff
@@ -70,7 +76,7 @@ func main() {
 			fmt.Println("PASS")
 			if *flVerbose {
 				for _, r := range vr {
-					if !r.Pass {
+					if r.Pass {
```

Comment:
Heh, sorry I missed this while reviewing #167.  But we can probably drop this check entirely, since we only hit this code when `fail == 0`.